nixio/multi_tag.py

```diff
@@ -52,7 +52,14 @@ def positions(self, da):
             raise TypeError("MultiTag.positions cannot be None.")
         if "positions" in self._h5group:
             del self._h5group["positions"]
-        self._h5group.create_link(da, "positions")
+        pos = da
+        if not isinstance(da, DataArray):
+            blk = self._parent
+            name = "f{self.name}-positions"
```

Comment:
Bad f-string.  It should be `f"{self.name}...`. 

More importantly, f-strings aren't supported in 2.7, so let's keep it Python2 friendly until after v1.5.